command/agent/event_endpoint.go

```diff
@@ -17,6 +17,103 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
+func (s *HTTPServer) EventSinksRequest(resp http.ResponseWriter, req *http.Request) (interface{}, error) {
+	if req.Method != http.MethodGet {
+		return nil, CodedError(405, ErrInvalidMethod)
+	}
+
+	args := structs.EventSinkListRequest{}
```

Comment:
hyper nit: using `:=` here, but `var` below for instantiating default value structs